roles/b2bua/templates/asterisk-extensions.conf.j2

```diff
@@ -1,2 +1,2 @@
 [globals](+)
-XIVO_AGID_IP = {{ engine_api_host }}
\ No newline at end of file
+WAZO_AGID_IP = {{ engine_api_host }}
```

Comment:
@fblackburn1 I changed this template but I'd like a second opinion on the implications on such a change.

The configuration file that is installed on the stack already includes `XIVO_AGID_IP` to avoid breaking custom dialplan that could potentially use wazo-agid.

I'm also unsure why we have or need this template

Reply:
Yeah this file is wrong...
- Improve the `when` clause to do not write this file if the value is the default value (i.e. localhost)
- Add migration script to remove this file (ex: https://github.com/wazo-platform/wazo-dist-upgrade/blob/master/bin/wazo-dist-upgrade-buster#L66-L69)
- Add upgrade note about this variable

So we won't take care of this file in future, and we will rely only on upgrade note for those using special value

Reply:
https://github.com/wazo-platform/wazo-upgrade/pull/86